types/src/front/lib/connectors_api.ts

```diff
@@ -121,6 +121,16 @@ export type ContentNodeWithParentIds = ContentNode & {
   parentInternalIds: string[] | null;
 };
 
+/**
+ * This type represents the ID that should be passed as parentId to a content node to hide it from the UI.
+ * This behavior is typically used to hide content nodes whose position in the ContentNodeTree cannot be resolved at time of upsertion.
+ */
+export const HiddenContentNodeParentId = "__dust_synchronizing__";
```

Comment:
Lets keep this connector specific until we need a broad support 🙏

Reply:
makes sense, done!